apps/web/components/getting-started/steps-views/SetupAvailability.tsx

```diff
@@ -72,8 +111,17 @@ const SetupAvailability = (props: ISetupAvailabilityProps) => {
             });
           }
 
-          mutation.mutate({ completedOnboarding: true });
+          await Promise.all(
+            DEFAULT_EVENT_TYPES.map(async (event) => {
+              return createEventType.mutate(event);
+            })
+          );
+
+          await utils.viewer.me.refetch();
```

Comment:
why do we need refetch?

Reply:
I copied it from source file, but I've just check its not necessary so I removed it :)